frontend/src/containers/MasterWindow.js

```diff
@@ -188,11 +188,13 @@ class MasterWindowContainer extends PureComponent {
       sortingOrder = (ordering.ascending ? '+' : '-') + ordering.fieldName;
     }
 
-    updateTabLayout(windowType, activeTabId).then(() => {
-      getTabRequest(activeTabId, windowType, docId, sortingOrder).then((rows) =>
-        updateTabTableData(tableId, rows)
-      );
-    });
+    updateTabLayout(windowType, activeTabId)
+      .then(() => {
+        getTabRequest(activeTabId, windowType, docId, sortingOrder).then(
+          (rows) => updateTabTableData(tableId, rows)
+        );
+      })
+      .catch((error) => error);
```

Comment:
So we're catching this error and then what ? Was it breaking the app or was reported by QA ? Maybe at least console.error it so that it will be of any use to us ?

Reply:
This was not breaking anything it was just outputing the err to console. This should be recorded in logstash normally and we should not pollute the console. Will add a todo for that